src/components/EkoVideo/EkoVideo.jsx

```diff
@@ -113,7 +115,7 @@ export function EkoVideo({
             clientSideParams,
             env,
             params,
-            events: eventList,
+            events: [...eventList, ...DEFAULT_EVENTS],
```

Comment:
I wonder if we should make sure this would be an array with unique values?
Not sure what would be the implications of having duplicates here...

Reply:
Actually, nevermind, I see that `eko-js-sdk` takes care of making sure the values are unique.